src/App.tsx

```diff
@@ -1,24 +1,66 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import { useEffect } from 'react';
+import { useTodoContext } from './context/TodoContext';
+import { getTodos } from './api/todos';
+import { Error } from './types/Error';
+import { TodoList } from './components/TodoList';
+import { TodoForm } from './components/TodoForm';
+import { FooterFilter } from './components/FooterFilter';
+import { ErrorMessage } from './components/ErrorMessage';
 
-const USER_ID = 0;
+const USER_ID = 10407;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const {
+    todos,
+    error,
+    setTodos,
+    setError,
+  } = useTodoContext();
+
+  const loadTodos = async () => {
+    try {
+      setError(null);
+      const todosFromServer = await getTodos(USER_ID);
+
+      setTodos(todosFromServer);
+    } catch {
+      setError(Error.LOAD);
+    }
+  };
+
+  useEffect(() => {
+    loadTodos();
+  }, []);
 
   return (
-    <section className="section container">
-      <p className="title is-4">
-        Copy all you need from the prev task:
-        <br />
-        <a href="https://github.com/mate-academy/react_todo-app-loading-todos#react-todo-app-load-todos">React Todo App - Load Todos</a>
-      </p>
-
-      <p className="subtitle">Styles are already copied</p>
-    </section>
+    <div className="todoapp">
+      <h1 className="todoapp__title">todos</h1>
+
+      <div className="todoapp__content">
+        <header className="todoapp__header">
+          <button type="button" className="todoapp__toggle-all active" />
```

Comment:
Add an 'onClick' event handler to the 'todoapp__toggle-all' button to handle the functionality of toggling all todos.

Reply:
this should be done in the next task, in that task I already have this logic